src/components/index.js

```diff
@@ -4,7 +4,6 @@ require('./cursor');
 require('./geometry');
 require('./light');
 require('./look-at');
-require('./look-controls');
```

Comment:
Done – this does leave a conflict on the `wasd-controls`name. Use `keyboard-controls` for the new component instead, on the assumption we might include arrow key support and/or custom bindings?